rmcmd.go

```diff
@@ -29,22 +32,23 @@ or all of the containers on a system with the -all flag.`,
 }
 
 func (c *rmcmd) RegisterFlags(fl *flag.FlagSet) {
-	fl.BoolVar(&c.all, "all", false, "Remove all sail containers.")
+	fl.BoolVar(&c.all, "all", false, "Remove all Sail containers.")
+	fl.BoolVar(&c.withData, "with-data", false, "Remove all data associated with the Sail instance.")
 }
 
 func (c *rmcmd) Run(fl *flag.FlagSet) {
-	c.gf.ensureDockerDaemon()
-
 	c.repoArg = fl.Arg(0)
 
 	if c.repoArg == "" && !c.all {
 		fl.Usage()
 		os.Exit(1)
 	}
 
-	names := c.getRemovalList()
+	// The docker daemon should not be running for the usage command.
```

Comment:
This comment doesn't make any sense

Reply:
I deleted the comment, but my thought process is as follows:

The docker daemon should not be required when it will not be used.